server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -564,9 +561,11 @@ public TSExecuteStatementResp executeQueryStatement(TSExecuteStatementReq req) {
       return RpcUtils.getTSExecuteStatementResp(TSStatusCode.SQL_PARSE_ERROR,
           ERROR_PARSING_SQL + e.getMessage());
     } catch (SQLParserException e) {
-      logger.error(CHECK_METADATA_ERROR, e);
-      return RpcUtils.getTSExecuteStatementResp(
-          TSStatusCode.METADATA_ERROR, CHECK_METADATA_ERROR + e.getMessage());
+      logger.info(ERROR_PARSING_SQL, req.getStatement() + " " + e.getMessage());
```

Comment:
is this log level too high for Server side?

Reply:
Lowered to debug.